braket_features/Allocating_Qubits_on_QPU_Devices.ipynb

```diff
@@ -10,7 +10,7 @@
     "\n",
```

Comment:
<p>Use the qubits -&gt; Using the qubits</p><p> </p><p>For the first sentence: Additionally, the device properties include calibration data, which you can use to find the qubits and qubit pairs with the highest fidelities for particular gates.</p><p> </p><p>For the second: ..., which can be any of the gates native to the Rigetti device, namely CPHASE, XY or CZ.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>